src/ir_Mitsubishi.cpp

```diff
@@ -867,11 +865,6 @@ bool IRrecv::decodeMitsubishi136(decode_results *results, uint16_t offset,
 #endif  // DECODE_MITSUBISHI136
 
 // Code to emulate Mitsubishi 136bit A/C IR remote control unit.
-//
-// Equipment it seems compatible with:
-//   Brand: Mitsubishi Electric,  Model: PEAD-RP71JAA Ducted A/C
-//   Brand: Mitsubishi Electric,  Model: 001CP T7WE10714 remote
```

Comment:
Again, we are losing info.
Slightly different here though.
Update .h file by adding the protocol name after the supported device.
e.g.
```
//   Brand: Mitsubishi Electric,  Model: PEAD-RP71JAA Ducted A/C (MITSUBISHI136)
```